decorators.go

```diff
@@ -242,6 +248,72 @@ func (CommandWithFlagsDecorator) Decorate(_ *Ecdysis, cmd *cobra.Command, c Comm
 	return nil
 }
 
+// -- PARSING CONFIGURATION --------------------------------------------------------------------
+
+// CommandWithConfiguration can be implemented by a command to parsing configuration.
+type CommandWithConfiguration interface {
+	Command
+
+	ParseConfig() Config
```

Comment:
OK, so the `Parse` in `ParseConfig` is not meant to be a verb (in the sense: this method parses a config), but a description of the config (a parse config)?

Reply:
@hariso the way I looked at it was the intention of this action (imperative). `ParseConfig`.

Reply:
Gotcha. In that case, it's slightly confusing, because having `ParseConfig` in `CommandWithConfiguration` indicates to me that `CommandWithConfiguration` is parsing a config. I might have also become too nitpicky in naming things.:)

Reply:
LGTM!